src/DiskIO/DiskThreads/DiskThreadsDiskFile.cc

```diff
@@ -189,7 +189,7 @@ DiskThreadsDiskFile::close()
         ioRequestor->closeCompleted();
         return;
     } else {
-        debugs(79, DBG_CRITICAL, HERE << "DiskThreadsDiskFile::close: " <<
+        debugs(79, DBG_CRITICAL, "DiskThreadsDiskFile::close: " <<
```

Comment:
"Who are you and what have you done with our Alex?"

Seriously though. When kinkie presented a script to do exactly these same HERE replacements, it was you vetoing its application with your insistence on a "perfection or nothing" policy to acceptable changes. Now you are accepting blatantly malformed code lines just because a script left it that way. So please, explain what is behind this complete reversal in your arguments?